asserts/snap_asserts.go

```diff
@@ -152,10 +152,11 @@ func assembleSnapBuild(assert assertionBase) (Assertion, error) {
 }
 
 // SnapRevision holds a snap-revision assertion, which is a statement by the
-// store acknowledging the receipt of a snap build and labeling it with a snap
-// revision.
+// store acknowledging the receipt of a build of snap-id and labeling it with a
```

Comment:
The documentation was right before here. It was the snap that was built, not the snap-id.